internal/nix/nix.go

```diff
@@ -169,6 +170,29 @@ func Version() (string, error) {
 	return version, nil
 }
 
+var nixPlatforms = []string{
+	"aarch64-darwin",
+	"aarch64-linux",
+	"i686-linux",
+	"x86_64-darwin",
+	"x86_64-linux",
+	// not technically supported, but should work?
+	// ref. https://nixos.wiki/wiki/Nix_on_ARM
+	// ref. https://github.com/jetpack-io/devbox/pull/1300
+	"armv7l-linux",
+}
+
+// EnsureValidPlatform returns an error if the platform is not supported by nix.
+// https://nixos.org/manual/nix/stable/installation/supported-platforms.html
+func EnsureValidPlatform(platform string) error {
+	for _, p := range nixPlatforms {
```

Comment:
`lo.Contains` ? or just use a map.

Reply:
I find current version simpler to write and read